.circleci/config.yml

```diff
@@ -217,7 +217,7 @@ jobs:
           destination: Builds
       - save_cache:
           name: Store library
-          key: library-{{ .Branch }}
+          key: library-2020-3-{{ .Branch }}-{{ epoch }}
```

Comment:
Same as line 71

Reply:
Epoch is 100% necessary here. If we don't put this the Library cache will not be regenerated when we push new assets on the same branch.